app/models/notification.rb

```diff
@@ -1,5 +1,7 @@
 class Notification < ActiveRecord::Base
 #  serialize :departments, Array
+  has_paper_trail
```

Comment:
again, `gem 'paper_trail'` isn't in the gemfile so don't see how this can work. 
also, seems inapplicable to this pr.... but it appears this pr is just built on #417 so...